test/test-connect-callback.js

```diff
@@ -0,0 +1,76 @@
+var assert = require('assert');
+var fs = require('fs');
+
+var unix = require('../lib/unix_dgram');
+var SOCKNAME = '/tmp/unix_dgram.sock';
+
+var sentCount = 0;
+var seenCount = 0;
+var expected = 300;
+
+process.on('exit', function() {
+  assert.equal(seenCount, sentCount);
+});
+
+try { fs.unlinkSync(SOCKNAME); } catch (e) { /* swallow */ }
+
+var server = unix.createSocket('unix_dgram', function(buf, rinfo) {
+  console.error('server recv', '' + buf, arguments);
+  assert.equal('' + buf, 'PING' + seenCount);
```

Comment:
Yes, I know. It was already noisy, but now that we're sending 300 messages I think it's too much.